lastpass-cli/plan.sh

```diff
@@ -2,7 +2,7 @@ pkg_name=lastpass-cli
 pkg_origin=core
 pkg_version=1.3.3
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
-pkg_license=("Apache-2.0")
+pkg_license=("GPL-2+")
```

Comment:
Could you change this to `GPL-2.0-or-later` to match the short identifier listed on the SPDX license site: https://spdx.org/licenses/GPL-2.0-or-later.html

Reply:
Done